assets/styles/style.css

```diff
@@ -0,0 +1,514 @@
+}
+
+.my-contact-form {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.my-contact-form input,
+textarea {
+  width: 100%;
+  padding: 15px 16px;
+  margin-bottom: 24px;
+  border: 2px solid #d0d9d4 !important;
+  border-radius: 4px;
+}
+
+.my-contact-form input[type="text"]:focus,
+input[type="email"]:focus,
+textarea:focus {
+  border: 2px solid #36b37e;
+  outline: none;
+}
+
+input:invalid,
+textarea:invalid {
+  border: 2px solid red;
+}
+
+.mobile-form {
+  width: 100%;
+  height: auto;
+}
+
+@media screen and (max-width: 640px) {
+  .desktop-nav {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .welcome {
+    background-image: url("../images/header-llustration-desktop.svg");
+    justify-content: center;
+    padding: 20%;
+    height: 100vh;
+    background-position: top center;
+  }
+
+  .about-me {
+    background-image: url("../images/about-desktop.png");
+    padding: 0 10%;
+    background-position: -126px 0;
+  }
+
+  .cards-showcase {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    column-gap: 24px;
+    row-gap: 24px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .navbar {
+    position: fixed;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    background-color: rgb(255 255 255 / 80%);
+  }
+
+  .navbar .site-title {
+    display: none;
+  }
+
+  .navbar .desktop-nav ul {
+    list-style-type: none;
+    text-align: center;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .navbar .desktop-nav ul li a {
+    padding: 10px;
+    text-decoration: none;
+    color: #172b4d;
+  }
+
+  .navbar .desktop-nav ul li a:hover {
+    color: #36b37e;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .desktop-form {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .mobile-form {
+    display: none;
+  }
+
+  .desktop-form fieldset {
+    display: flex;
+    gap: 24px;
+    border: none;
+    padding: 0;
+    width: 100%;
+  }
+
+  .cards-showcase {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    column-gap: 24px;
+    row-gap: 24px;
+  }
+
+  .about-me {
+    background-image: url("../images/about-desktop.png");
+    padding: 0 10%;
+    background-position: top center;
+  }
+
+  .about-text {
+    padding: 0 20%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact-me {
+    background-image: url("../images/footer-desktop.png");
+    padding: 0 10% 225px;
+    display: grid;
+    grid-template-columns: 40% 60%;
+    align-items: start;
+    column-gap: 24px;
+    background-position: bottom center;
+    object-fit: cover;
+  }
+
+  .contact-me .section-heading {
+    font-size: 30px;
+  }
+
+  .social-links-footer {
+    border-top: 1px solid #505f79;
+    padding: 5px;
+  }
```

Comment:
Kindly adjust the color of the footer line, as displayed in Figma 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199645119-84cc5d66-bb81-4a1f-9743-49bccf3d9cb7.png)  | ![image](https://user-images.githubusercontent.com/42799579/199645094-d12dbbe3-f9e3-4220-a222-bebae4d37246.png)  |

Also, please notice that this is a different element, not a border-top from the social links.
Tip: you can use a `hr` element 😄.